ISISPostProcessRPM/rpmbuild/autoreduce-mq/usr/bin/AutoreductionProcessor/autoreduction_logging_setup.py

```diff
@@ -1,7 +1,7 @@
 import logging.handlers
 
-LOGGING_LEVEL = logging.INFO
-LOGGING_LOC = '/var/log/autoreduction.log'
+LOGGING_LEVEL = logging.DEBUG
```

Comment:
Hi Jamie, From memory setting activemq logging to DEBUG generates an enormous amount of logging. By default this to be logging.INFO on production since there may be a risk that running it with logging.DEBUG for too long period may have side effect